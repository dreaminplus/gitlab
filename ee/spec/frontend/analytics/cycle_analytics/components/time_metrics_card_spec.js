import { mount } from '@vue/test-utils';
import TimeMetricsCard from 'ee/analytics/cycle_analytics/components/time_metrics_card.vue';
import { group } from '../mock_data';

describe('TimeMetricsCard', () => {
  const { full_path: groupPath } = group;
  let wrapper;

  const createComponent = () => {
    return mount(TimeMetricsCard, {
      propsData: {
        groupPath,
      },
    });
  };

  beforeEach(() => {
    wrapper = createComponent();
  });

  afterEach(() => {
    wrapper.destroy();
  });

  it('matches the snapshot', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
