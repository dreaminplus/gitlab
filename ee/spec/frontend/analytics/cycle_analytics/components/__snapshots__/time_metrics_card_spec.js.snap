// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimeMetricsCard matches the snapshot 1`] = `
<div
  class="card"
>
  <!---->
  <div
    class="card-header"
  >
    <strong>
      Time
    </strong>
  </div>
  <div
    class="card-body"
  >
    <!---->
    <!---->
     
    <div
      class="d-flex"
    >
      <div
        class="js-metric-card-item flex-grow text-center"
      >
        <h3
          class="my-2"
        >
          2 days
        </h3>
         
        <p
          class="text-secondary gl-font-sm mb-2"
        >
          Lead time
        </p>
      </div>
      <div
        class="js-metric-card-item flex-grow text-center"
      >
        <h3
          class="my-2"
        >
          1.5 days
        </h3>
         
        <p
          class="text-secondary gl-font-sm mb-2"
        >
          Cycle time
        </p>
      </div>
    </div>
     
  </div>
  <!---->
  <!---->
</div>
`;
